specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-11-15/services.json

```diff
@@ -0,0 +1,642 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB",
+    "description": "Azure Cosmos DB Database Service Resource Provider REST API",
+    "version": "2022-11-15"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/services": {
+      "get": {
+        "x-ms-examples": {
+          "CosmosDBServicesList": {
+            "$ref": "./examples/CosmosDBServicesList.json"
+          }
+        },
+        "operationId": "Service_List",
+        "description": "Gets the status of service.",
+        "parameters": [
```

Comment:
Can v4 common types be used in case it's not a breaking change?

Reply:
V4 mostly has changes to support RPaaS RPs. We recommend you going with the latest version possible.

Reply:
Noted. We'll take this up in the next API version changes.